.circleci/config.yml

```diff
@@ -364,22 +364,22 @@ jobs:
       - run:
           name: Generate SQL content
           command: |
-            mkdir /tmp/private-generated-sql
-            cp -r sql/ /tmp/private-generated-sql/sql
+            mkdir /tmp/generated-sql
+            cp -r sql/ /tmp/generated-sql/private-sql
```

Comment:
I'd prefer to keep the public and private paths more parallel, so that we aren't introducing another layer of terminology here.

How about this:

- We use `/tmp/workspace` as the root
- Public sql ends up under `/tmp/workspace/generated-sql/sql`
- Private sql ends up under `/tmp/workspace/private-generated-sql/sql`

That way the paths we're using here mirror the branches where the content will end up.

Reply:
Sounds good!